src/components/ReviewAnswers.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Button, Card } from 'react-bootstrap';
+
+const getAnswersClass = (question, userAnswers, answer) => {
+  if (answer.isCorrect) {
+    return 'text-success';
+  } else if (userAnswers[question.id] === answer.id) {
+    return 'text-danger';
+  } else {
+    return '';
+  }
+};
+
+const ReviewAnswers = ({
+  score,
+  questions,
+  quizName,
+  userAnswers,
+  numOfQuestions,
+}) => {
+  return (
+    <div>
+      <div>
+        <h3>
+          Your score is : {score} / {numOfQuestions}
```

Comment:
```suggestion
          Your score is: {score} / {numOfQuestions}
```
`:`s do not have spaces before them